### DISCUSSION TITLE
Get x-rate-limit-reset on rate-limit error?

### DISCUSSION THREAD

**Discussion by clashen70** [noweh/twitter-api-v2-php#48] [Q&A]
When posting a Tweet with $client->tweet()->create()->performRequest returns with rate-limit error, there is no information on the x-rate-limit-reset time in the error message?

Is there a way to find the value for x-rate-limit-reset when you are getting rate-limit error for posting a tweet?

**Comment by noweh** [comment]
This part of the README file should interest you: https://github.com/noweh/twitter-api-v2-php#include-the-http-headers-provided-by-twitter-in-the-response

**Reply by clashen70** [reply]
It is indeed interesting. When performRequest() throws exception the $response object never gets any value, and hence no header-information can be requested from the response, or? How do you reach or print what is described as "output:" in this part of the README file?

**Reply by noweh** [reply]
Unfortunately this is only possible before the exception, while results are still available
